18_regexp_lab/01.py

```diff
@@ -10,5 +10,7 @@
 with open(kuku.path, 'r') as f:
     for line in f:
         if kuku.key in line:
-            res = re.search("=\s?(\w*)", line)
+            if re.search('^\s?#', line):
+                continue
+            res = re.search("=\s+(\w*)", line)
             print(res.group(1))

```

Comment:
nope. If I have several spaces before the # then code and I run  with `-k name` then I get 'many'.
```
     # name = many spaces 2
```